website/source/docs/job-specification/reschedule.html.md

```diff
@@ -59,6 +62,25 @@ job "docs" {
    number of reschedule happen within it. If more than `attempts` number of
    failures happen with this interval, Nomad will not reschedule any more.
 
+- `delay` `(string: "5s")` - Specifies the duration to wait before attempting
+  to reschedule a failed task. This is specified using a label suffix like "30s" or "1h".
+
+- `delay_function` `(string: <varies>)` - Specifies the function according to which
+  the initial delay specified in `delay` changes. `delay_function` has three possible
+  values which are described below.
+    - `linear` - The delay between reschedule attempts stays constant at the `delay` value defined above.
+    - `exponential` - The delay between reschedule attempts doubles until it reaches a specified
+      `max_delay` value.
+    - `fibonacci` - The delay between reschedule attempts is calculated by adding the two most recent
+      delays applied. For example if `delay` is set to 5 seconds, the next five reschedule attempts  will be
+      delayed by 5 seconds, 5 seconds, 10 seconds, 15 seconds, and 25 seconds respectively.
+
+- `max_delay` `(string: <varies>)` - is an upper bound on the delay beyond which it will not increase. This parameter
```

Comment:
whats the default?

Reply:
defaults vary by job type

Reply:
are they documented anywhere?

Reply:
Its documented right below in another section in this file :)